app/models/parcel_item.rb

```diff
@@ -70,6 +66,11 @@ class ParcelItem < Ekylibre::Record::Base
   has_one :delivery, through: :parcel
   has_one :storage, through: :parcel
   has_one :contract, through: :parcel
+  has_one :product_enjoyment, class_name: "ProductEnjoyment", as: :originator, dependent: :destroy
```

Comment:
You can remove the "class_name" argument as the relations are already named like the classes they link to.